<launch>
    <node pkg="Unitree_A1" name="A1_control" type="A1_main" output="screen"/> <!-- imu串口读取发送节点 -->
    <node pkg="Unitree_A1" name="A1_serial" type="serial_send.py" output="screen" respawn="true"/> <!-- imu串口读取发送节点 -->
    
</launch>
<!-- 
pkg： 节点所在的功能包名称
type: 节点的可执行文件名称
name: 节点运行时的名称
output  =“log | screen” (可选)，日志发送目标，可以设置为 log 日志文件，或 screen 屏幕,默认是 log
respawn =“true | false” (可选)，如果节点退出，是否自动重启
required=“true | false” (可选)，该节点是否必须，如果为 true,那么如果该节点退出，将杀死整个 roslaunch
ns=“xxx” (可选)，在指定命名空间 xxx 中启动节点
machine=“机器名”，在指定机器上启动节点
args=“xxx xxx xxx” (可选)，将参数传递给节点
-->